### ISSUE TITLE
Add css animations

### ISSUE THREAD

**Issue by MikelTafalla** [bcantello/peoples_voice_project#67]
We could think a few places where that can be cool, like landing page or the contact form. I was thinking something like this, but the other way around.
Like everything is written and then the info inputed gets into the envelop when clicking submit and the envelop closes and dissapears to show that the message is sent
https://codepen.io/palimadra/pen/noVwzy

**Comment by bcantello** [comment]
Yes. However each animation should be its own ticket with specific instructions of what the animation should do
